src/backend/aspen/workflows/import_pango_lineages/import.py

```diff
@@ -0,0 +1,198 @@
+    Intended to be temporary, should drop by end of this script.
+    copy-pasta from import_gisaid/save.py"""
+    metadata = MetaData()
+    suffix = re.sub("-", "", str(uuid.uuid1()))
+    table_name = f"{source_table.name}_{suffix}"
+    cols = []
+    for col in source_table.columns:
+        cols.append(
+            Column(
+                col.name, col.type, primary_key=col.primary_key, nullable=col.nullable
+            )
+        )
+
+    table_object = Table(table_name, metadata, *cols)
+    session.execute(CreateTable(table_object))
+    return table_object
+
+
+def mv_table_contents(session, source_table, dest_table):
+    """Deletes contents of dest, copies in contents from source to dest.
+    copy-pasta from import_gisaid/save.py"""
+    cols = [col.name for col in dest_table.columns]
+    session.execute(dest_table.delete())
+    session.execute(dest_table.insert().from_select(cols, source_table.select()))
+
+
+def drop_table(session, table_obj):
+    """Drops table. Intended to be used on temporary table only.
+    copy-pasta from import_gisaid/save.py"""
+    session.execute(DropTable(table_obj))
+
+
+def load_lineages_data(lineages: list[str]) -> None:
+    """Loads all the lineages into DB.
+
+    Approach to this is basically duplicating what's in
+        backend/aspen/workflows/import_gisaid/save.py
+    Idea is to load all the data into a temp table with same structure,
+    then once all loaded, drop the rows from "real" table, move the new data
+    over from the temp table, and finally drop the temp table to wrap up.
+
+    Original GISAID import deals with a lot more data, so that has some
+    performance-specific bits that are not included in this version.
+    """
+    LINEAGE_COL_NAME = "lineage"
+
+    interface: SqlAlchemyInterface = init_db(get_db_uri(Config()))
+    with session_scope(interface) as session:
+
+        dest_table = PangoLineages.__table__
+        temp_table = create_temp_table(session, dest_table)
+
+        # Load data into temp_table
+        lineage_objects = [{LINEAGE_COL_NAME: lineage} for lineage in lineages]
+        session.execute(temp_table.insert(), lineage_objects)
+
+        # Replace previous data with new data from temp_table
+        mv_table_contents(session, temp_table, dest_table)
+        drop_table(session, temp_table)
+
+        # Final sanity check before we commit
+        count_db_lineages = session.query(dest_table).count()
+        print(f"Imported {count_db_lineages} lineage rows")
+        if len(lineages) != count_db_lineages:
+            raise RuntimeError("Something went wrong loading DB. Abort!")
+            # This exception will bubble up, end session, cause rollback.
+
+        session.commit()
+
+
+@click.command()
+@click.option(
+    "pango_lineages_file",
+    "--lineages-file",
+    type=click.File("r"),
+    required=True,
+    help="Pango lineages file to parse and import.",
+)
+@click.option(
+    "--parse-without-import",
```

Comment:
I know all the other workflow Python scripts have a `test` flag for doing smoke tests that goes like this:
```
@click.option("--test", type=bool, is_flag=True)

... later in main code block ...

    if test:
        print("Success!")
        return  # end script early

```

Should I be doing it as well? Or since there's another testing approach via `--parse-without-import`, it's fine without the smoke test version?

Reply:
This is fine. Basically we just have those --test flags to do the barest-minimum spot checks of our scripts that don't have better tests (😢) in CI - do they compile? Do all their imports work? So it might be nice to make sure this is wired into the `make check-images` target in the root `Makefile`